src/test_merge_sort.py

```diff
@@ -0,0 +1,26 @@
+"""Tests for merge sort."""
+from merge_sort import merge_sort
+
+import pytest
+
+
```

Comment:
Good job parametrizing this.  I think you left out a few important cases, however, like the empty list, the list with one thing, and larger lists that you could easily build with randint.